streakerbar/AppDelegate.swift

```diff
@@ -31,8 +31,15 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         
         statusItem.image = icon
         statusItem.menu = statusMenu
-        var timer = NSTimer.scheduledTimerWithTimeInterval(0.4, target: self, selector: Selector("randShit"), userInfo: nil, repeats: true)
-        getGithubUsernameFromGitconfig()
+        updateTitle("?")
+//        var timer = NSTimer.scheduledTimerWithTimeInterval(0.4, target: self, selector: Selector("randShit"), userInfo: nil, repeats: true)
+        if let username = getGithubUsernameFromGitconfig() {
+            updateStatusForUser(username)
+        } else if let username = promptForUsername() {
+            updateStatusForUser(username)
+        } else {
+            // :( 
+        }
```

Comment:
feel like `updateStatusForUser` could be put on a timer. Maybe not 0.4 seconds, but 5 min or something.